logo-full.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 479.02 117.54"><defs><style>.cls-1{font-size:48px;fill:#2684ff;}.cls-1,.cls-2{font-family:Circular-Bold, Circular;font-weight:700;}.cls-2{font-size:69px;}.cls-2,.cls-5{fill:#253858;}.cls-3{letter-spacing:0.01em;}.cls-4{letter-spacing:0em;}</style></defs><title>logo-full</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><text class="cls-1" transform="translate(100.91 41.02)">REFINED</text><text class="cls-2" transform="translate(98.91 100.02)">BITBUCK<tspan class="cls-3" x="296.42" y="0">E</tspan><tspan class="cls-4" x="337.4" y="0">T</tspan></text><path class="cls-5" d="M0,5.52,16.34,100H68.55L86.8,5.52ZM58.46,85H27.19l-9.78-64.5h52Z"/></g></g></svg>
```

Comment:
I noticed though that the typography in the SVG doesn't look like the original image you posted, can you take a look at that?

**Original** 
![original](https://user-images.githubusercontent.com/7514993/36848848-d54daa02-1d38-11e8-93db-541545c932ee.png)

**This one**
![svg](https://user-images.githubusercontent.com/7514993/36848817-c0cadf0a-1d38-11e8-89dc-337a7bcf0a68.png)

Reply:
Oops - I forgot to convert the text to outlines!